user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_LDAP_settings.md

```diff
@@ -45,15 +45,15 @@ To configure custom LDAP settings in DataMiner Cube:
    - **Phone**: The user’s fixed phone number.
    - **Mobile**: The user’s mobile phone number.
    - **Pager**: The user’s pager number.
-   - **Filter**: The LDAP search filter to find all users. Note that in XML ampersands must be encoded as "&amp;".
+   - **Filter**: The LDAP search filter to find all users. Note that in XML ampersands must be encoded as "&amp;amp;".
```

Comment:
If we enclose the text in backticks, it will also be rendered correctly, but the source will be easier to read:
```suggestion
   - **Filter**: The LDAP search filter to find all users. Note that in XML ampersands must be encoded as `&amp;`.
```